sbt-compatibility/src/sbt-test/sbt-compatibility/library/test

```diff
@@ -3,45 +3,37 @@
 
 # NOT OK to introduce source or binary incompatibilities
 # during 0.1.x
--> v0-1-1/mimaReportBinaryIssues
```

Comment:
Why do you remove them, eventually?

Reply:
I had initially added them to debug the test, and check that the problem came from binary compatibility itself. We already have [`run-mima`](https://github.com/scalacenter/sbt-compatibility/tree/master/src/sbt-test/sbt-compatibility/run-mima) checking that `mimaReportBinaryIssues` is indeed run by `compatibilityCheck`, so the explicit calls to `mimaReportBinaryIssues` are noise here I think.